pkg/antctl/raw/multicluster/create/clusterset.go

```diff
@@ -0,0 +1,141 @@
+	}
+	if o.secret != "" && o.leaderClusterServer == "" {
+		return fmt.Errorf("the leader cluster server is required in member cluster")
+	}
+
+	return nil
+}
+
+func NewClusterSetCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "clusterset",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Create a ClusterSet in a leader or member cluster",
+		Long:    "Create a ClusterSet in a leader or member cluster",
+		Example: clusterSetExamples,
+		RunE:    clusterSetRunE,
+	}
+
+	o := &clusterSetOptions{}
+	clusterSetOpt = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterSet")
+	command.Flags().StringVarP(&o.leaderCluster, "leader-cluster-id", "", "", "leader cluster ID of the ClusterSet")
+	command.Flags().StringVarP(&o.leaderClusterServer, "leader-server", "", "", "leader cluster server address of the ClusterSet")
+	command.Flags().StringVarP(&o.leaderClusterNamespace, "leader-namespace", "", "", "the Namespace where Antrea Multi-cluster Controller is running in leader cluster")
+	command.Flags().StringToStringVarP(&o.memberClusters, "member-clusters", "", nil, "cluster ID and ServiceAccount group of the member clusters(e.g. --member-clusters member1=sa1,member2=sa2)")
+	command.Flags().StringVarP(&o.secret, "secret", "", "", "Secret to access the leader cluster, it is required only when creating ClusterSet in member cluster")
+
+	return command
+}
+
+func clusterSetRunE(cmd *cobra.Command, args []string) error {
+	if err := clusterSetOpt.validateAndComplete(); err != nil {
+		return err
+	}
+	if len(args) != 1 {
+		return fmt.Errorf("exactly one NAME is required, got %d", len(args))
+	}
+
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: multiclusterscheme.Scheme})
+	if err != nil {
+		return err
+	}
+	clusterSetName := args[0]
+	clusterSet := &multiclusterv1alpha1.ClusterSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      clusterSetName,
+			Namespace: clusterSetOpt.namespace,
+		},
+		Spec: multiclusterv1alpha1.ClusterSetSpec{
+			Leaders: []multiclusterv1alpha1.MemberCluster{
+				{
+					ClusterID: clusterSetOpt.leaderCluster,
+					Secret:    clusterSetOpt.secret,
+					Server:    fmt.Sprintf("https://%s", strings.Replace(clusterSetOpt.leaderClusterServer, "https://", "", 1)),
+				},
+			},
+			Namespace: clusterSetOpt.leaderClusterNamespace,
+		},
+	}
+
+	for memberCluster, serviceAccount := range clusterSetOpt.memberClusters {
+		clusterSet.Spec.Members = append(clusterSet.Spec.Members, multiclusterv1alpha1.MemberCluster{
+			ClusterID:      memberCluster,
+			ServiceAccount: serviceAccount,
+		})
+	}
+
+	if err := k8sClient.Create(context.TODO(), clusterSet); err != nil {
+		return err
+	}
+	fmt.Fprintln(cmd.OutOrStdout(), fmt.Sprintf(`ClusterSet "%s" created`, clusterSetName))
```

Comment:
Unify the message? some commands use "xxx is created" while some use "xxx created"

Reply:
changed "is created" to "created", as well as "delete"